src/app/components/MediaLoader/index.styles.ts

```diff
@@ -27,29 +28,45 @@ export default {
 
   landscapeFigure: () => css({ aspectRatio: '16 / 9' }),
   portraitFigure:
-    (isEmbedded = false) =>
-    ({ mq }: Theme) => [
+    (pageType: string, isEmbedded = false) =>
+    ({ mq, spacings }: Theme) => [
       css({
         aspectRatio: '9 / 16',
         display: 'flex',
         flexDirection: 'column',
-        ...(!isEmbedded && {
-          maxWidth: `${pixelsToRem(185)}rem`,
-          [mq.GROUP_1_ONLY]: {
-            maxWidth: `${pixelsToRem(256)}rem`,
-          },
-          [mq.GROUP_2_ONLY]: {
-            maxWidth: `${pixelsToRem(274)}rem`,
-          },
+        ...(!isEmbedded &&
+          pageType !== MEDIA_ARTICLE_PAGE && {
+            maxWidth: `${pixelsToRem(185)}rem`,
+            [mq.GROUP_1_ONLY]: {
+              maxWidth: `${pixelsToRem(256)}rem`,
+            },
+            [mq.GROUP_2_ONLY]: {
+              maxWidth: `${pixelsToRem(274)}rem`,
+            },
+            [mq.GROUP_3_ONLY]: {
+              maxWidth: `${pixelsToRem(200)}rem`,
+            },
+            [mq.GROUP_4_MIN_WIDTH]: {
+              maxWidth: `${pixelsToRem(190)}rem`,
+            },
+          }),
+        ...(pageType === MEDIA_ARTICLE_PAGE && {
```

Comment:
I was expecting this styling to be part of the MediaArticlePage styles, and not the MediaLoader component itself (but I could be wrong - I am not 100% how the styling for the media loader is supposed to work) . Is this an anti-pattern? Wonder what @HarveyPeachey and @amoore108 think?

Reply:
I'd personally prefer to keep page specific styling as close to that page as possible.

The MediaLoader component itself should be fairly generic and styled by a parent element/stylesheet if the styles need to be overridden.

Reply:
We do have the `className` prop already, which is used on the `figure` element: https://github.com/bbc/simorgh/blob/54418e9d180be71a4074f967227e8a9f0b491e28/src/app/components/MediaLoader/index.tsx#L262

So a CSS style could be passed in from the MediaArticlePage to override the styles.

Reply:
Yeah classname is your friend here, I agree